rtloader/test/rtloader/rtloader_test.go

```diff
@@ -38,9 +42,15 @@ func TestGetPyInfo(t *testing.T) {
 	if path == "" {
 		t.Errorf("Python path is null")
 	}
+
+	// Check for leaks
+	helpers.AssertMemoryUsage(t)
```

Comment:
Note: You might move this line inside `TestMain` as you did in `rtloader/test/aggregator/aggregator.go`. This comment can be also applied to other test files.

Reply:
I do not have this in `TestMain` for the aggregator tests. Maybe I'm missing something? Or maybe you reviewed commit by commit?